app-backend/batch/src/main/scala/com/azavea/rf/batch/Main.scala

```diff
@@ -16,7 +16,15 @@ import com.azavea.rf.batch.sentinel2.ImportSentinel2
 import com.azavea.rf.batch.stac.{ReadStacFeature}
 import com.azavea.rf.batch.notification.NotifyIngestStatus
 
+import cats.effect.IO
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+
 object Main {
+
+  implicit val contextShift = IO.contextShift(global)
```

Comment:
I think without this in scope cats recommends just making an `IOApp`, so that might be safer / more future-proof